CondTools/DT/src/DTStatusFlagHandler.cc

```diff
@@ -125,8 +125,10 @@ void DTStatusFlagHandler::getNewObjects() {
   cond::Time_t snc = runNumber;
   if (runNumber > last)
```

Comment:
@cms-sw/db-l2 can this check be moved to the start of the function?

Reply:
to the extend that it may matter, DT DPG is fine with the proposal